### PR TITLE
fixes after 2022.04

### PR DESCRIPTION
**Pull request by antgonza**
General improvements after deployment.

### CODE REVIEW

File: qiita_db/artifact.py [hunk 1]
```diff
@@ -79,17 +99,6 @@ def iter_by_visibility(cls, visibility):
             for a_id in qdb.sql_connection.TRN.execute_fetchflatten():
                 yield cls(a_id)
 
-    @classmethod
```
Comment (antgonza) [inline qiita_db/artifact.py:82]: This is not used anywhere in the code.

File: qiita_db/artifact.py [hunk 2]
```diff
@@ -615,12 +624,15 @@ def delete(cls, artifact_id):
                 qdb.util.move_filepaths_to_upload_folder(
                     study.id, filepaths)
 
-                sql = """UPDATE qiita.prep_template
-                         SET artifact_id = NULL
-                         WHERE prep_template_id IN %s"""
-                qdb.sql_connection.TRN.add(
-                    sql, [tuple(pt.id for a in all_artifacts
-                                for pt in a.prep_templates)])
+                # there are cases that an artifact would not be linked to a
```
Comment (antgonza) [inline qiita_db/artifact.py:627]: Now we have artifacts (like those generated by the sequencing processing pipeline) without prep/analysis linkages.

File: qiita_pet/templates/sitebase.html [hunk 3]
```diff
@@ -532,11 +532,7 @@
                   </li>
                   <li role="separator" class="divider"></li>
                   <li>
-                    {% if qiita_sha=="" %}
-                      <a href="#">
-                    {% else %}
-                      <a href="https://github.com/biocore/qiita/commit/{{qiita_sha}}">
-                    {% end %}
+                    <a href="https://github.com/qiita-spots/qiita/releases">
```
Comment (antgonza) [inline qiita_pet/templates/sitebase.html:535]: We should point to the releases page vs. the `sha` that is too dev oriented.

File: qiita_ware/commands.py [hunk 4]
```diff
@@ -43,11 +43,8 @@ def _ssh_session(p_url, private_key):
     """
     scheme = p_url.scheme
     hostname = p_url.hostname
-    # if port is '' Python 2.7.6 will raise an error
-    try:
-        port = p_url.port
-    except Exception:
-        port = 22
+
```
Comment (antgonza) [inline qiita_ware/commands.py:46]: The ssh connection is currently broken due to newer python, this fixes it.

**Review by charles-cowart — APPROVED** [review]
Everything looks good so far.

**Pull request merged**